Source/Layout/ASYogaUtilities.mm

```diff
@@ -106,6 +106,10 @@ float yogaFloatForCGFloat(CGFloat value)
   }
 }
 
+float cgFloatForYogaFloat(float yogaFloat) {
```

Comment:
nit: brace style in Texture has open braces for functions and methods on a newline.